chall08/xtang.c

```diff
@@ -0,0 +1,42 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   xtang.c                                            :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: xtang <xtang@student.hive.fi>              +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2020/05/05 11:30:48 by xtang             #+#    #+#             */
+/*   Updated: 2020/05/05 15:59:39 by xtang            ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+int	ft_ie_except_after_c(char *str)
+{
+	int i;
+
+	i = 0;
+	while (str[i] != '\0' || str[i] != 'e')
+	{
+		if (str[i] == '\0')
+			return (1);
+		if (str[i] == 'e')
+			break ;
+		i++;
+	}
+	i = 0;
+	while (str[i] != '\0')
+	{
+		if (str[i] == 'e')
```

Comment:
I like how you check == 'e' and use that to check for "ie" and "ei" at the same time. Clever.